src/libraries/System.Formats.Tar/src/System/Formats/Tar/TarWriter.cs

```diff
@@ -33,17 +43,16 @@ public TarWriter(Stream archiveStream, IEnumerable<KeyValuePair<string, string>>
         }
 
         /// <summary>
-        /// Initializes a <see cref="TarWriter"/> instance that can write tar entries to the specified stream, optionally leave the stream open upon disposal of this instance, and can specify the format of the underlying archive.
+        /// Initializes a <see cref="TarWriter"/> instance that can write tar entries to the specified stream, optionally leaves the stream open upon disposal of this instance, and can optionally specify the preferred default format when writing entries using the <see cref="WriteEntry(string, string?)"/> method.
```

Comment:
Chopping some of these really long lines will add goodness.  But I also have a technical comment:

> and can optionally specify the preferred default format when writing entries using the <see cref="WriteEntry(string, string?)"/> method.

That's not a "preference", right, it's just "the format to use with `WriteEntry(string, string)`"  which means "preferred default" should just be deleted.

Reply:
Fixed, both length of lines and the wording.

Reply:
The line seems to have actually gotten longer.  Definitely way longer than my laptop can read:

![image](https://user-images.githubusercontent.com/10642668/174413598-bba3dec0-c727-43fa-8932-1c4ae43e8ad0.png)

Reply:
I'm also still unclear on anything that says `default format`.  It's not a default, in that it will always be that value (unless you're expecting some feature where, e.g. a block device is written in V7 format even if the "default" format says to be GNU).

Looking at the code, it's not a default, it's a declaration.  So all instances of "default format" and "preferred default format" should just be "format".

Reply:
Fixed